### ISSUE TITLE
Suddenly Peninsula doesn´t work anymore

### ISSUE THREAD

**Issue by warmup72** [Celve/Peninsula#24]
Today Peninsula crashes and after restart macOS didn´t want to start it anymore?
Cache and Prefs are deleted but it doesn't work on my Mac :-(
![Screenshot_2024-12-15 15 31 44_e0nOdd](https://github.com/user-attachments/assets/98dcd611-df8d-41f4-92fd-689ae578906f)

**Comment by Celve** [comment]
Seems like I have to do some codesign. I will figure it out recently.

**Comment by Celve** [comment]
Does the following one work for you? https://github.com/Celve/Peninsula/releases/tag/v0.0.6-alpha

**Comment by warmup72** [comment]
Yes, the 0.0.6 works - I have to confirm it in the control panel and then it works!

**Comment by Celve** [comment]
> Yes, the 0.0.6 works - I have to confirm it in the control panel and then it works!

I will get a Developer ID recently and at that time you don't need to do this step.

**Comment by Celve** [comment]
> I really like Peninsula - for me it's a great first approach to a great tool In my opinion there are still a few little things to make it more and more perfect. I can tell you more about this when it works again :-)
> 
> Thank you

Thank you! I’d love to hear your thoughts on how we can improve it further.

**Comment by warmup72** [comment]
Yes you are right - set to English it works

**Comment by Celve** [comment]
Ok, so I know that it's language issue. I will change my system lang to German to figure it out.

**Comment by warmup72** [comment]
No sorry - it was the default setting "Follow System" which doesn´t work

**Comment by Celve** [comment]
> No sorry - it was the default setting "Follow System" which doesn´t work

So as long as you change lang settings in Peninsula then the notification works?
